### ISSUE TITLE
StanfordCorefResolver throws IOException

### ISSUE THREAD

**Issue by nreimers** [dkpro/dkpro-core#752]
When using the StanfordCoreferenceResolver from DKPro-1.8.0-SNAPSHOT, I get the attached exception.
[error.txt](https://github.com/dkpro/dkpro-core/files/81004/error.txt)

Caused by: java.io.IOException: edu.stanford.nlp.io.RuntimeIOException: java.io.IOException: Unable to resolve "de/tudarmstadt/ukp/dkpro/core/stanfordnlp/lib/coref/en/default/gender.map.ser.gz" as either class path, filename or URL

I think this is due to line 196 in the [StanfordCoreferenceResolver.java](
https://github.com/dkpro/dkpro-core/blob/master/dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordCoreferenceResolver.java)

Line 196 is:
props.setProperty(Constants.GENDER_NUMBER_PROP, logicalBase + "gender.map.ser.gz");, 

I would imagine that it must be (replace logicalBase by base):
props.setProperty(Constants.GENDER_NUMBER_PROP, base + "gender.map.ser.gz");

I was not able to test ist. So if someone could test and fix that, that would be great.

**Comment by reckart** [comment]
Looks like there is a problem with bintray. The model file there appears to be broken:

https://bintray.com/bintray/jcenter/edu.stanford.nlp%3Astanford-corenlp/3.5.2/view#files/edu/stanford/nlp/stanford-corenlp/3.5.2

-> stanford-corenlp-3.5.2-models.jar	8 months ago	3 MB

http://search.maven.org/#artifactdetails%7Cedu.stanford.nlp%7Cstanford-corenlp%7C3.5.2%7Cjar

-> stanford-corenlp-3.5.2-models.jar	04-May-2015	321.7 M

**Comment by reckart** [comment]
@jbaruch Since I didn't find an issue tracker for JCenter/Bintray, I'm trying to ping you this way: We've recently added Bintray as a source for artifacts in DKPro Core, but apparently there are some artifacts not properly mirrored from Maven Central:

The Stanford CoreNLP models JAR appears to be broken on JCenter:

https://bintray.com/bintray/jcenter/edu.stanford.nlp%3Astanford-corenlp/3.5.2/view#files/edu/stanford/nlp/stanford-corenlp/3.5.2

-> stanford-corenlp-3.5.2-models.jar 8 months ago 3 MB

http://search.maven.org/#artifactdetails%7Cedu.stanford.nlp%7Cstanford-corenlp%7C3.5.2%7Cjar

-> stanford-corenlp-3.5.2-models.jar 04-May-2015 321.7 M

If there is a better way to report such problems, please tell us.

**Comment by reckart** [comment]
@nreimers the problem with StanfordCoreferenceResolver seems unrelated to the broken models jar. I just removed the dependency (only required for StanfordSentimentAnalyzer) and could run the StanfordCoreferenceResolverTest locally. Please check if the you have the JARs for

```
de.tudarmstadt.ukp.dkpro.core.stanfordnlp-model-coref-en-default 20150420.1
de.tudarmstadt.ukp.dkpro.core.stanfordnlp-upstream-coref-en-default 20150420
```

on your classpath. Please refer to the [DKPro User Guide](https://dkpro.github.io/dkpro-core/documentation/) (currently available only under the 1.8.0-SNAPSHOT header) section "Adding components as dependencies (Maven)" for details on how to add model dependencies to your project.

**Comment by nreimers** [comment]
Adding the models
`de.tudarmstadt.ukp.dkpro.core.stanfordnlp-model-coref-en-default 20150420.1
de.tudarmstadt.ukp.dkpro.core.stanfordnlp-upstream-coref-en-default 20150420`

to the pom.xml solved the problem, thanks!
